guarddog/analyzer/metadata/npm/__init__.py

```diff
@@ -0,0 +1,16 @@
+from guarddog.analyzer.metadata.npm.empty_information import NPMEmptyInfoDetector
+from guarddog.analyzer.metadata.npm.potentially_compromised_email_domain import \
+    NPMPotentiallyCompromisedEmailDomainDetector
+from guarddog.analyzer.metadata.npm.release_zero import NPMReleaseZeroDetector
+
+NPM_METADATA_RULES = {}
+
+classes = [
```

Comment:
How about finding them automatically based on inheritance of a common base class `NpmMetadataDetector`? https://www.studytonight.com/python-howtos/how-to-find-all-the-subclasses-of-a-class-given-its-name

It would make it easier to add new heuristics and make sure we don't forget to include them